plume-models/src/apps.rs

```diff
@@ -0,0 +1,80 @@
+use canapi::{Error, Provider};
```

Comment:
should i be watching the canapi repo as well?

Reply:
As you want